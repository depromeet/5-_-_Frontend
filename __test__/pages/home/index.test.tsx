import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Home from '@/pages/home';

describe('<Home />', () => {
  // <Home/> 컴포넌트에 'Test one' text가 있는지 확인하는 test
  it('renders a heading', () => {
    const { container } = render(<Home />);

    const home = screen.getByText('Test one');

    expect(home).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
