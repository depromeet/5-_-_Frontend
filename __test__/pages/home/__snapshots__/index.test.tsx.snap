// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders a heading 1`] = `
<div>
  <span
    class="css-1vg6q84 emfg3hs0"
  >
    Test one
  </span>
  <span
    class="css-kaq1dv emfg3hs1"
  >
    Test two
  </span>
</div>
`;
